ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -676,6 +714,8 @@ const ControlledTierCard = (props: {profile: Profile, spinnerProps: WithSpinnerO
                           text={`${institutionDisplayName} must allow you to access ${ctDisplayName} data`}/>
       {displayEraCommons &&
          <ControlledTierEraModule profile={profile} eligible={isEligible} spinnerProps={spinnerProps}/>}
+      {unsafeAllowSelfBypass &&
```

Comment:
for someone unfamiliar with why we did this, it may be a little surprising for selfBypass to be the condition for displaying this.  Renaming the component could also help to make this more clear.

Reply:
Updated the component to SelfBypassCTComplianceTrainingModule